asset/script/main.js

```diff
@@ -54,6 +56,56 @@
         }
     };
 
+    function contextOutputControl (element) {
+
+      $(document).on('lint-results', function( event, results) {
+          var inputEl = $(document).find('#post-input')[0];
```

Comment:
It's actually InputEl (as in InputElement) - but happy to write that out fully. @antislice